src/main/java/com/devonfw/ide/sonarqube/common/impl/config/ConfigurationFactory.java

```diff
@@ -74,11 +77,13 @@ private static File findConfigFile(File folder) {
     if (folder == null) {
       return null;
     }
+
     File configFile = new File(folder, Configuration.ARCHITECTURE_JSON);
     if (configFile.exists()) {
       return configFile;
     }
-    return findConfigFile(folder.getParentFile());
+
+    return findConfigFile(folder.getParentFile().getAbsoluteFile());
```

Comment:
Actually the even more clean and correct fix would be to do `getAbsoluteFile()` only once in line 56 like this:
```
File configFile = findConfigFile(fileToScan.getAbsoluteFile().getParentFile());
```

Reply:
Makes sense. I thought that I would have to call getAbsoluteFile() at every recursive call, but even better that it works like this